api/graphql/schema.py

```diff
@@ -122,7 +122,7 @@ async def load_families_for_participants(
 ) -> list[list['Family']]:
     flayer = FamilyLayer(connection)
     fam_map = await flayer.get_families_by_participants(participant_ids=participant_ids)
-    return [fam_map.get(p) for p in participant_ids]
+    return [fam_map.get(p) for p in participant_ids if p in fam_map] or [[]]
```

Comment:
The dataloaders loaders here want you to always return the correct amount of elements, so a filter on a map here can break the loader, I presume the default should just be an empty list from the .get.

```suggestion
    return [fam_map.get(p, []) for p in participant_ids]
```